game-warhammer40kdarktide/index.js

```diff
@@ -37,7 +41,12 @@ const tools = [
   },
 ];
 
+// Not sure if there is a more elegant way to get this for patching later
+let GAME_PATH = null;

```

Comment:
We have a `queryPath` util function defined in here, does it solve this?

Reply:
It might. My reasoning on that is that from what I could tell, queryPath is an Async function, and I wanted to make sure that the unpatch process was handled synchronously, since it's in will-purge and the patcher is actively being deleted next.

But if I misjudged that, then sure, it can be simpler in both places, and there's probably a better place to store that path either way.